modules/core/server/controllers/mongoose-filtering.server.controller.js

```diff
@@ -0,0 +1,54 @@
+﻿// Service for managing Filtering/Parameterized-Queries.
+
+var _query = null;
+var _modifiedQuery = null;
+var _totalDocumentCount = null;
+var _page = 0;
+var _take = 50; // default to prevent unreasonable take() requests
```

Comment:
I just re-read your comment. I didn't understand it correctly before. To be clear, you would like the "parameters" (take, & page) to be accessible in the URL? The API route `/api/parameterized-query/articles` is a POST since we could be sending complex data that we can't put in the query-string. I can pull the *take*, and *page* out of the request body & put them into the URL as query-string parameters.

Is that what you were driving at?